apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -130,33 +140,35 @@ export class FilesStorageUC {
 		await this.authorizationService.checkPermissionByReferences(userId, allowedType, parentId, context);
 	}
 
-	async download(userId: EntityId, params: DownloadFileParams) {
+	private checkFileName(entity: FileRecord, params: DownloadFileParams): void | NotFoundException {
+		if (entity.name !== params.fileName) {
+			throw new NotFoundException('File not found', `${FilesStorageUC.name}:download`);
+		}
+	}
+
+	private checkScanStatus(entity: FileRecord): void | NotAcceptableException {
+		if (entity.securityCheck.status === ScanStatus.BLOCKED) {
+			throw new NotAcceptableException('File is blocked', `${FilesStorageUC.name}:download`);
```

Comment:
```suggestion
			throw new NotAcceptableException('FILE_IS_BLOCKED', `${FilesStorageUC.name}:download`);
```

Reply:
Why do you want to write it all uppercase? I'm especially irritated in connection with the new enum for ErrorType, which introduces error constants in all uppercase. Does this have to do something with that?